### PR TITLE
toggle sinequa theme on click

### PR DESCRIPTION
**Pull request by hebus**
* Toggle Sinequa theme selection

### CODE REVIEW

File: projects/vanilla/src/app/search/search.component.ts [hunk 1]
```diff
@@ -18,7 +18,8 @@ import { BsFacetDate } from '@sinequa/analytics/timeline';
 @Component({
   selector: 'app-search',
   templateUrl: './search.component.html',
-  styleUrls: ['./search.component.scss']
+  styleUrls: ['./search.component.scss'],
+  encapsulation: ViewEncapsulation.None
```
Comment (ericleib) [inline projects/vanilla/src/app/search/search.component.ts:22]: Maybe using ::ng-deep in the CSS could have less potential side-effects?

File: projects/vanilla/src/styles/app.scss [hunk 2]
```diff
@@ -64,8 +63,41 @@ $action-menu-breakpoint: sm;
...
+    @import "../../../../node_modules/@sinequa/components/theme/vanilla";
+}
+
 @import 'themes/custom';
+// allow cusomization
+// @import 'themes/cyberpunk';
+
 
+// Facets -- TO REMOVE WHEN CSS WILL BE IN THE COMPONENT
+sq-facet-card {
```
Comment (ericleib) [inline projects/vanilla/src/styles/app.scss]: I think it can be removed if you update the version of the sinequa libraries in the package.json ?

**Review by ericleib — APPROVED** [review]

File: projects/vanilla/src/app/search/search.component.ts (hunk 1 above)
Comment (hebus) [reply projects/vanilla/src/app/search/search.component.ts:22]: not needed here, the `ViewEncapsulation.None` do the job and in the `scss` file, only css style about `seach-component` is set

File: projects/vanilla/src/styles/app.scss (hunk 2 above)
Comment (hebus) [reply projects/vanilla/src/styles/app.scss]: I'will do this

Comment (hebus) [reply projects/vanilla/src/styles/app.scss]: removed

**Review by ericleib — APPROVED** [review]

**Pull request merged**